README.aspnet.md

```diff
@@ -0,0 +1,107 @@
+﻿# Featured Tags
+
+* `4.8`
+  * `docker pull mcr.microsoft.com/dotnet/framework/aspnet:4.8`
+* `3.5`
+  * `docker pull mcr.microsoft.com/dotnet/framework/aspnet:3.5`
+
+# What is ASP.NET
+
+ASP.NET is a high productivity framework for building Web Applications using Web Forms, MVC, Web API and SignalR.
+
+This repository contains `Dockerfile` definitions for ASP.NET Docker images. These images use the [IIS image](https://hub.docker.com/r/microsoft/iis/) as their base.
```

Comment:
The statement about using the IIS base image is incorrect.

Reply:
removed